src/effects/VST/VSTEffect.h

```diff
@@ -342,24 +342,24 @@ class VSTEffect final
    ResourceHandle mResource;
 #endif
 
-   AEffect *mAEffect;
+   AEffect* mAEffect{};
 
    VstTimeInfo mTimeInfo;
 
-   bool mUseLatency;
-   int mBufferDelay;
+   bool mUseLatency{true};
+   int mBufferDelay{0};
 
-   unsigned mBlockSize;
+   unsigned mBlockSize{8192};
```

Comment:
This first commit is good.

Do you know that you could even write
```
unsigned mBlockSize{ mUserBlockSize };
```

The in-class initializer doesn't have to be a constant expression, and it can refer to earlier declared members.  That would more exactly preserve the mening of the assignment

```
mBlockSize = mUserBlockSize = 8192;
```

Reply:
This would look better - done, although I had to use a static cast:

`unsigned mBlockSize{ static_cast<unsigned>(mUserBlockSize) };`

Reply:
Maybe mBlockSize should just be size_t.

Or you can use the = syntax not {} .  There is a difference.  {} disallows narrowing conversions.

Reply:
I opted to make mBlockSize a size_t